test/lib/sync-fs.spec.js

```diff
@@ -1,6 +1,13 @@
 const assert = require('chai').assert;
 
+const chai = require('chai');
 const fs = require('../../src/lib/sync-fs');
+// NOTE: be careful with the names, this one is the original fs module
+// from nodeJS where the upper one is from our sync-fs module
+const fs_ = require('fs');
+const sinon = require('sinon');
+
+const { expect } = chai;
```

Comment:
Let's just combine this with the `require`:

```js
const { expect } = require('chai');
```

Also, we try not to mix `expect` and `assert` in the same files.  The go-forward pattern is to use `expect`, so can you just remove `assert` and update that test to use `expect`? (Just in this file...)

Reply:
Updated.

Reply:
:point_up: Looks like maybe this did not get updated yet!

Reply:
Updated, now. :)